src/test/librados_test_stub/TestClassHandler.cc

```diff
@@ -43,7 +45,9 @@ void TestClassHandler::open_class(const std::string& name,
 void TestClassHandler::open_all_classes() {
   assert(m_class_handles.empty());
 
-  DIR *dir = ::opendir(".libs");
+  const char* env = getenv("CEPH_LIB");
+  std::string CEPH_LIB(env ? env : "lib");
+  DIR *dir = ::opendir(CEPH_LIB.c_str());
   if (dir == NULL) {
```

Comment:
@alimaredia @cbodley  Now if CEPH_LIB env is not set, "lib" is used instead of previously used ".libs". Was it intentionally?
```
% ./unittest_rbd_mirror               
2016-04-18 09:10:23.392541 7fb9f8fdeb00 -1 WARNING: the following dangerous and experimental features are enabled: *
[==========] Running 58 tests from 11 test cases.
[----------] Global test environment set-up.
[----------] 1 test from TestMockImageReplayer
[ RUN      ] TestMockImageReplayer.Blah
test/librados_test_stub/TestClassHandler.cc: In function 'void librados::TestClassHandler::open_all_classes()' thread 7fb9f8fdeb00 time 2016-04-18 09:10:23.500613
test/librados_test_stub/TestClassHandler.cc: 52: FAILED assert(false)
```